gulp-core-build-karma/src/KarmaTask.ts

```diff
@@ -67,6 +78,30 @@ export class KarmaTask extends GulpTask<IKarmaTaskConfig> {
 
       completeCallback();
     } else {
+      // Normalize the match expression if one was specified
+      const { testMatch } = this.taskConfig; // tslint:disable-line:typedef
+      if (testMatch) {
+        let normalizedMatch: RegExp;
+        if (typeof testMatch === 'string') {
+          normalizedMatch = new RegExp(testMatch as string);
+        } else if (testMatch instanceof RegExp) {
+          normalizedMatch = testMatch;
+        } else {
+          completeCallback('The testMatch regular expression is invalid');
+          return;
+        }
+
+        // tslint:disable:max-line-length
+        const testsJsFileContents: string = [
+          `var context = require.context('${path.posix.join('..', this.buildConfig.libFolder)}', true, ${normalizedMatch.toString()});`,
+          `context.keys().forEach(context);`,
```

Comment:
So we're just writing a tests.js file instead of telling karma about the individual files? Is that what we want? That won't let someone run a single test.